src/script.ts

```diff
@@ -14,38 +15,53 @@ import {
 
 async function main() {
   const client = new MendixPlatformClient();
-
-  const app = client.getApp(input.AppId);
-  const repo = app.getRepository();
-  const workingCopy = await app.createTemporaryWorkingCopy(input.BranchName);
+  const app = client.getApp(input.AppID);
+  //const repo = app.getRepository();
+  //console.log(await repo.getBranches());
+  const workingCopy = await app.createTemporaryWorkingCopy(input.BranchLine);
   const model = await workingCopy.openModel();
-  const domainModel = await getOrCreateDomainModel(model);
 
-  const objectsFolder = getOrCreateFolder(
-    domainModel.containerAsModule,
-    "objects"
-  );
 
-  let x = 0;
-  let y = 0;
-  for (var ent of input.Entities) {
-    const newEnt = getOrCreateEntity(domainModel, ent.Name, x, y);
-    x += 100;
-    y += 100;
+  for (var mod of input.Modules) {
+    let x = 0;
+    let y = 0;
+    const domainModel = await getOrCreateDomainModel(model, mod.Name);
+    const objectsFolder = getOrCreateFolder(
+      domainModel.containerAsModule,
+      "objects"
+    );
+    const pagesFolder = getOrCreateFolder(
+      domainModel.containerAsModule,
+      "pages"
+    );
+    const resourcesFolder = getOrCreateFolder(
+      domainModel.containerAsModule,
+      "resources"
+    );
+    for (let ent of mod.Entitys) {
+      const newEnt = getOrCreateEntity(domainModel, ent.Name, x, y);
+      x += 100;
+      y += 100;
+      for (let att of ent.Attributes) {
+        const type = PrimitiveType[att._Type as keyof typeof PrimitiveType]
+        if (type) {
```

Comment:
You can use a ternary operation here: 

`getOrCreateAttribute(newEnt, att.Name, type ? type : PrimitiveType.STRING)`

I even think you can simplify this to:

`getOrCreateAttribute(newEnt, att.Name, type || PrimitiveType.STRING)`